extensions/target-specific/chromium/win-d3d11-angletextures/extension.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "win-d3d11-angletextures",
```

Comment:
nit: please use the same naming format as the other extensions (WinD3D11AngleTextures)

Reply:
I think I am consistent with gfx-type-helpers, ax-tree, chromium-helpers. In cases where the words are merged they refer to class names in code ?

Reply:
Both "gfx-type-helpers" and "chromium-helpers" are headless extensions (won't be directly visible to the user), so those are fine. For AXTree, though the file is called "ax-tree", the extension name is "AXTree" and that's what's shown on the JsDbg landing page. Please follow the same style.